tekton/README.md

```diff
@@ -72,44 +73,47 @@ access).
 
 To run the `publish-tekton-pipelines` `Task` and create a release:
 
-1. Pick the revision you want to release and replace the value of the `PipelineResouce`
-   [`tekton-pipelines` `revision`](publish-run.yaml#11), e.g.:
+1. Pick the revision you want to release and replace the value of the
+   `PipelineResouce` [`tekton-pipelines` `revision`](publish-run.yaml#11), e.g.:
```

Comment:
Not caused by this PR but there's a typo here, I can open a PR to fix it after this is merged or maybe someone with rights can push a commit onto the current PR.

```suggestion
   `PipelineResource` [`tekton-pipelines` `revision`](publish-run.yaml#11), e.g.:
```

Reply:
See https://github.com/tektoncd/pipeline/pull/693.